src/dialogs/alwaysonbotDialogs/UpdateProfile/updateProfileStep.ts

```diff
@@ -0,0 +1,81 @@
+import {
+    Choice, ChoicePrompt, ComponentDialog, DialogTurnResult, PromptValidatorContext, TextPrompt,
+    WaterfallDialog,
+    WaterfallStepContext
+} from "botbuilder-dialogs";
+import { CommonPromptValidatorModel } from "../../../models/commonPromptValidatorModel";
+import { ContinueAndFeedbackStep } from "../Common/continueAndFeedbackStep";
+import { FeedBackStep, FEED_BACK_STEP } from "../Common/feedBackStep";
+import { UpdateAddressStep, UPDATE_ADDRESS_STEP } from "./UpdateAddress/updateAddressStep";
+import { CommonChoiceCheckStep, COMMON_CHOICE_CHECK_STEP } from "./UpdatePhoneNumber/commonChoiceCheckStep";
+import { UPDATE_PHONE_NUMBER_STEP,UpdateMyPhoneStep } from "./UpdatePhoneNumber/updateMyPhoneStep";
+import i18n from "../../locales/i18nconfig";
```

Comment:
should be i18nConfig

Reply:
we would like to have further discussion on this

Reply:
../../locales/i18nconfig what i mean is this file name should be i18nConfig. not import **i18nConfig** from ../../locales/i18nconfig